p256/src/arithmetic/scalar.rs

```diff
@@ -813,4 +814,16 @@ mod tests {
             U256::from_u8(4),
         );
     }
+
+    #[test]
+    fn batch_invert() {
+        let scalars: [Scalar; 10] =
+            core::array::from_fn(|_| NonZeroScalar::try_from_rng(&mut OsRng).unwrap().into());
+
+        let inverted_scalars = Scalar::batch_invert(scalars.as_slice()).unwrap();
+
+        for (scalar, inverted_scalar) in scalars.into_iter().zip(inverted_scalars) {
+            assert_eq!(inverted_scalar, scalar.invert().unwrap());
+        }
+    }
```

Comment:
I guess these are fine for now. It'd be good to get them in `primefield::test_field_invert` eventually but the `OsRng` dependency is a complication. It would be good if it used `proptest` instead for randomized testing, but similarly `primefield` isn't set up for that.

Maybe add some TODO comments to that effect? (move to `primefield::test_field_invert`, switch to `proptest`)